rpms/python-kombu/python-kombu.spec

```diff
@@ -58,6 +59,7 @@ also provide proven and tested solutions to common messaging problems.
 %patch1 -p1
 %patch2 -p1
 %patch3 -p1
+%patch4 -p1
```

Comment:
This works just fine and I don't think it's worth changing, but just so you know the ``%autosetup`` macro applies all listed patched automatically so you don't have to remember to add this line. For example, line 56 would be ``%autosetup -p1 -q -n %{srcname}-%{version}`` and lines 58-62 could go away.